Registry/RepositoryRegistry.php

```diff
@@ -46,6 +46,27 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
+    public function names()
```

Comment:
Actually wonder if this is worth it. This information could be retrieved from the array keys of `all()` - at the cost of instantiating the repositories. This could however be avoided if we had a lazy collection. /cc @WouterJ @dbu